package/gluon-client-bridge/luasrc/lib/gluon/upgrade/320-gluon-client-bridge-wireless

```diff
@@ -42,6 +38,73 @@ util.foreach_radio(uci, function(radio, index, config)
 		ifname = suffix and 'client' .. suffix,
 		disabled = disabled or false,
 	})
+end
+
+local function configure_owe(radio, index, config, radio_name)
+	local name_client = 'client_' .. radio_name
+	local name_owe = 'owe_' .. radio_name
+	local suffix = radio_name:match('^radio(%d+)$')
+
+	local ap = config.ap
+	local owe = config.owe
+
+	local disabled = uci:get_bool('wireless', name_client, 'disabled')
+
+	uci:delete('wireless', name_owe)
+	local macaddr = util.get_wlan_mac(uci, radio, index, 3)
+
+	if not owe() or not ap() or not macaddr then
+		return
+	end
+
+	uci:section('wireless', 'wifi-iface', name_owe, {
+		device = radio_name,
+		network = 'client',
+		mode = 'ap',
+		ssid = owe.ssid(),
+		macaddr = macaddr,
+		ifname = suffix and 'owe' .. suffix,
+		disabled = disabled or false,
+		encryption = 'owe',
+		mfp = 2,
```

Comment:
Shouldn't this vif be `hidden` if `transition_mode` was enabled?

Reply:
Happens here https://github.com/freifunk-gluon/gluon/pull/1945/files#diff-09ec2a20205df7d6051ebd606cf29f0dR94.

Reply:
This is done in the tm_setup function.